NgocMaiDang_democypress/cypress/e2e/addbook_spec.cy.js

```diff
@@ -0,0 +1,39 @@
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false
+});
+  
+describe('Add book with name "Git Pocket Guide" to my collection and verify it in my profile', () => {
+    it('Should add the book to the collection and verify it in the profile', () => {
+  
+      //Visit the login page and login
+      cy.visit('https://demoqa.com/login');
+      cy.get('#userName').type('ngocmai197');
+      cy.get('#password').type('Demoqa@ngocmai1907');
+      cy.get('#login').click();
+
+      //Go to the bookstore
+      cy.contains('Book Store').click();
```

Comment:
Should get before contains to make sure that there is no element with different tag but has the same text